site/data/ar/docs/custom-wallets.mdx

```diff
@@ -172,30 +172,30 @@ description: إنشاء محفظة مخصصة
       type: '{ getUri?: () => Promise<string> }',
       typeSimple: 'object',
       description:
-        'الدالة لحل رابط اتصال المحفظة الجوال',
+        'Function for resolving a mobile wallet connection URI',
     },
     {
       name: 'desktop',
       required: false,
       type: '{ getUri?: () => Promise<string> }',
       typeSimple: 'object',
-      description: 'الدالة لحل رابط التطبيق المكتبي العميق',
+      description: 'Function for resolving a desktop app deep link',
     },
     {
       name: 'qrCode',
       required: false,
-      type: "{ getUri: () => Promise<string>, instructions?: { learnMoreUrl: string, steps: Array<{ step: 'install' | 'create' | 'scan', title: string, description: string }}}}",
+      type: "{ getUri: () => Promise<string>, instructions?: { learnMoreUrl: string, steps: Array<{ step: 'install' | 'create' | 'scan', title: string, description: string }> }}}",
       typeSimple: 'object',
       description:
-        'كائن يحتوي على دالة لحل الرابط الرمزي الرباعي الأبعاد (QR) ، بالإضافة إلى تعليمات اختيارية لإعداد المحفظة الجوال',
+        'Object containing a function for resolving the QR code URI, plus optional mobile wallet setup instructions',
     },
     {
       name: 'extension',
       required: false,
-      type: "{ instructions?: { learnMoreUrl: string, steps: Array<{ step: 'install' | 'create' | 'scan', title: string, description: string }}}}",
+      type: "{ instructions?: { learnMoreUrl: string, steps: Array<{ step: 'install' | 'create' | 'scan', title: string, description: string }> }}}",
       typeSimple: 'object',
       description:
-        'كائن يحتوي على تعليمات اختيارية لإعداد الإضافة المتصفح',
+        'Object containing optional browser extension setup instructions',
     },
```

Comment:
Aren't these supposed to be arabic ?

Reply:
Nope, we don't translate `<PropTable>` elements